.github/CHANGELOG.md

```diff
@@ -2,6 +2,50 @@
 
 <h3>New features since last release</h3>
 
+* Custom gradient transforms can now be created using the new
+  `@qml.gradients.gradient_transform` decorator on a batch-tape transform.
```

Comment:
```suggestion
  `@qml.gradient_transform` decorator on a batch-tape transform.
```

This is accessible top-level right? (Or were you deliberately highlighting the module name here?)

Reply:
Oh... I just checked, and it isn't top level! That was deliberate though. What do you think?

Reply:
I think for consistency with stuff like `@qml.qfunc_transform`, it'd be good to have it top-level?

Reply:
I just gave it a go, and ran into non-stop circular import errors 😬 Let's leave it here for now, and we can always graduate it separately